docs/api/field_type_reference.md

```diff
@@ -412,6 +412,28 @@ $countryValue = new Country\Value(
 );
 ```
 
+!!! dxp
+
+    ## DAM Field Type
+
+    This Field Type stores asset information from a DAM system.
+
+    | Name   | Internal name | Expected input type |
+    |--------|---------------|---------------------|
+    | `DAM` | `dam`      | mixed             |
```

Comment:
```suggestion
    | `DAM` | `ezimageasset`      | mixed             |
```

Dam fieldtype is replacing ezimageasset, not sure what should be stated in terms of `name` then.

Reply:
For other Field Types the name is the one displayed in the Content Type creation UI, so ImageAsset as well I think